pkg/chartmuseum/server/multitenant/server_test.go

```diff
@@ -641,6 +659,14 @@ func (suite *MultiTenantServerTestSuite) TestBadChartUpload() {
 	suite.Equal(400, res.Status(), "400 POST /api/charts")
 }
 
+func (suite *MultiTenantServerTestSuite) TestSemver2Validation() {
```

Comment:
we should add a positive test here too, posting a chart with a valid semver to make sure it doesnt trigger the 400

Reply:
👌 I will add it